cms/envs/common.py

```diff
@@ -516,6 +516,15 @@
     #   in the LMS and CMS.
     # .. toggle_tickets: 'https://github.com/open-craft/edx-platform/pull/429'
     'DISABLE_UNENROLLMENT': False,
+
+    # .. toggle_name: FEATURES['DISABLE_ADVANCED_SETTINGS']
+    # .. toggle_implementation: DjangoSetting
+    # .. toggle_default: False
+    # .. toggle_description: Set to True to disable the advanced settings page in Studio for all users except staff.
```

Comment:
Can you clarify what is meant by 'staff' here? Is it django staff, or course staff?

Reply:
@pdpinch, done.